assets/site/guide.sass

```diff
@@ -142,15 +142,13 @@ html.cockpit-guide
 
   h2, h3, h4, h5, h6,
   dl.toc .part,
-  dl.toc .chapter,
     position: static
     padding: 1rem 0 0.5rem
 
-  h2,
```

Comment:
But this change here does not correspond to the change mentioned in the commit message.